vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -71,6 +71,7 @@
       <AdditionalIncludeDirectories>
         $(ReactNativeWindowsDir)stubs;
         $(ReactNativeWindowsDir)\Shared\tracing;
+        $(FollyDir)\..\..\fast_float-6.1.4\include;
```

Comment:
@jonthysell Ideally we'd like to change in path in line13 of CSSTokenizer to just be `#include <fast_float/fast_float.h`. However, after I added the directory to the AdditionalIncludeDirectories above, ReactCommon can't find the file. It can only find the file when I write out the full relative path in CSSTokenizer.h. Any ideas why that is? How come the AdditionalIncludeDirectories isn't working as expected?